shared/js/initApp.js

```diff
@@ -150,7 +150,37 @@ $(window).on('function_editor_closed', function() {
   }
 })(appOptions.level);
 
+dashboard.saveProject = function(callback) {
+  var app_id = dashboard.currentApp.id;
+  dashboard.currentApp.startBlocks = Blockly.Xml.domToText(Blockly.Xml.blockSpaceToDom(Blockly.mainBlockSpace));
+  if (app_id) {
+    storageApps().update(app_id, dashboard.currentApp, function(data) {
+      console.log('Updated!');
```

Comment:
Looks like we no-op `console.log` but not `warn` `error` `debug` `info` etc:

https://github.com/code-dot-org/code-dot-org/blob/1245758eca2a49e7d3c21ac51b8da528b8bda482/dashboard/app/assets/javascripts/application.js.erb#L48